code-reviews/recipes/azure-pipelines-yaml.md

```diff
@@ -0,0 +1,108 @@
+# Code Review for Azure Pipelines YAML Files
```

Comment:
The rest of the recipes here follow a "template" with the same naming and sections - could you look at that and re-structure this content to follow a similar format?